### PR TITLE
Remove all upgrade jobs to gke-cvm-1.8, and pin cvm-1.7 to 1.7.5

### PR DESCRIPTION
**Pull request by krzyzacy**
/assign @jlowdermilk @krousey 
cc @crimsonfaith91

### CODE REVIEW

File: prow/config.yaml [hunk 1]
```diff
@@ -11216,312 +11216,6 @@ periodics:
...
-      secret:
-        secretName: service-account
-    - name: ssh
-      secret:
-        defaultMode: 256
-        secretName: ssh-key-secret
-
-- interval: 2h
-  agent: kubernetes
   name: ci-kubernetes-e2e-gke-cvm-stable-cvm-master-upgrade-master
```
Comment (crimsonfaith91) [inline prow/config.yaml]: so we are not removing this test?

Comment (krzyzacy) [reply prow/config.yaml]: for now 1.6-cvm to 1.8-gci is still valid

Comment (crimsonfaith91) [reply prow/config.yaml]: noted, thanks

**Comment by krzyzacy** [comment]
Updated: Cleaned up everything that upgrades to cvm-1.8, and pin upgrade that from cvm-1.7 on gke to use 1.7.5 for now.

**Comment by krousey** [comment]
/lgtm
thanks for doing this

**Comment by krzyzacy** [comment]
(nobody is using `--extract=ci/v1.7.5` as of now, hopefully that syntax will still work)

**Comment by krousey** [comment]
Can't be worse than what is happening now.

**Pull request merged**